env-example

```diff
@@ -18,4 +18,4 @@ NEW_RELIC_LICENSE_KEY=
 
 # metrics
 CASTLE_HOST=
-CASTLE_TEST_EPISODE=
+CASTLE_TEST_PODCAST=
```

Comment:
I should update the README for this, but if we add the ability to select which podcast you want to look at metrics for soon, it shouldn't be around much longer anyway.